tests/test_bukuDb.py

```diff
@@ -766,13 +763,16 @@ def test_replace_tag(self):
                 self.assertEqual(from_db[3], parse_tags(["test,tes,est,__04"]))
 
     def test_tnyfy_url(self):
+        tny, full = 'http://tny.im/yt', 'https://www.google.com'
         # shorten a well-known url
-        shorturl = self.bdb.tnyfy_url(url="https://www.google.com", shorten=True)
-        self.assertEqual(shorturl, "http://tny.im/yt")
+        with mock_http(tny, status=200):
+            shorturl = self.bdb.tnyfy_url(url=full, shorten=True)
+        self.assertEqual(shorturl, tny)
```

Comment:
Mocking HTTP responses improves performance for tests (except for functional tests for bukuserver, due to most time there being spent on webpage rendering)